test/coverlet.core.tests/CoverageSummaryTests.cs

```diff
@@ -75,8 +75,52 @@ private void SetupData()
             _modules.Add("module", documents);
         }
 
+        private void SetupAditionalData()
```

Comment:
nit: rename `SetupDataForAverageCalculation` and call inside constructor `CoverageSummaryTests` one time.

Reply:
Rename is done,.... but if SetupDataForAverageCalculation is called into the constructor, previous tests will fail because initial data will have two modules.
I want to have two scnearios to test, one with one module(TestCalculateBranchCoverage_OneModule), and other with multiple modules (TestCalculateBranchCoverage_Modules).... what do u think?

Reply:
I prefer custom private fields with custom setup for tests for instance

```cs
private Modules _averageCalculationSingleModule;
private Modules _averageCalculationMultiModule;
```
and load in constructor.

Reply:
Changed... pls take a look at line 123 (LMK if its ok) => I want to reuse data from SetupDataSingleModule

Reply:
I see it's ok, some nits and LGTM